packages/manifold/contracts/lazyclaim/ERC1155LazyPayableClaim.sol

```diff
@@ -34,7 +34,8 @@ contract ERC1155LazyPayableClaim is IERC165, IERC1155LazyPayableClaim, ICreatorE
             interfaceId == type(IERC165).interfaceId;
     }
 
-    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2) LazyPayableClaim(initialOwner, delegationRegistry, delegationRegistryV2) {}
+    constructor(address initialOwner, address delegationRegistry, address delegationRegistryV2, uint256 mintFee,
```

Comment:
Do not pass it into constructor.  Default to a value and then set.  Otherwise we will not have consistent addresses across chains with different fees